ligolang/src/test/zinc_test.ml

```diff
@@ -57,15 +57,21 @@ type test =
   unit ->
   unit
 
-let expect_simple_compile_to ?reason:(enabled = false) ?(index = 0)
+type dialect = Reasonligo | Cameligo | Pascaligo | Jsligo
```

Comment:
Would you be able to use `Ligo_compile.Helpers.v_syntax` instead of making a new enum?

Reply:
Most likely :+1: